boxoffice/views/admin_discount.py

```diff
@@ -52,12 +62,38 @@ def jsonify_discount_policies(data_dict):
     )
 @render_with({'text/html': 'index.html', 'application/json': jsonify_discount_policies}, json=True)
 def admin_discount_policies(organization):
+    RESULTS_PER_PAGE = 6
     query = request.args.get('search')
-    if query:
-        discount_policies = DiscountPolicy.query.filter(DiscountPolicy.title.ilike('%{query}%'.format(query=query))).all()
+    try:
+        page = int(request.args.get('page', 1))
+    except ValueError:
+        page = 1
+
+    if request.is_xhr:
+        discount_policies = DiscountPolicy.query.filter(
+            DiscountPolicy.organization == organization
+        ).order_by('created_at desc')
+
+        if query:
+            discount_policies = discount_policies.filter(
+                DiscountPolicy.title.ilike('%{query}%'.format(query=query))
+            )
+
+        total_policies = discount_policies.count()
+
+        offset = (page - 1) * RESULTS_PER_PAGE
+        discount_policies = discount_policies.limit(RESULTS_PER_PAGE).offset(offset)
+
+        return dict(
+            org=organization, title=organization.title,
```

Comment:
For this https://github.com/hasgeek/boxoffice/blob/master/boxoffice/templates/index.html#L7

For different view, organization might not be present to use organization.title in the template
https://github.com/hasgeek/boxoffice/blob/master/boxoffice/views/admin_report.py#L22